tkey/src/androidTest/java/com/web3auth/tkey/tkeyServiceProviderTest.java

```diff
@@ -25,9 +26,9 @@ public class tkeyServiceProviderTest {
     public void provider() {
         try {
             PrivateKey key = PrivateKey.generate();
-            new ServiceProvider(false, key.hex,false, null,null,null,null,null);
+            new ServiceProvider(false, key.hex,false, null,null,null);
             System.gc();
-        } catch (RuntimeError e) {
+        } catch (RuntimeError | JSONException e) {
             fail(e.toString());
         }
```

Comment:
Test service provider creation where last 3 values are not null and use tss is true.